app/routes/wallet/dids.py

```diff
@@ -19,13 +19,38 @@
 router = APIRouter(prefix="/v1/wallet/dids", tags=["wallet"])
 
 
-@router.post("", response_model=DID)
+@router.post("", response_model=DID, summary="Create Local DID")
 async def create_did(
     did_create: Optional[DIDCreate] = None,
```

Comment:
Note: this model is our custom model. And when it's passed to the acapy_wallet service:
```py
        result = await acapy_wallet.create_did(
            did_create=did_create, controller=aries_controller
        )
```
that service is expecting an ACA-Py model. So for consistency, we should just modify our custom model, and ensure we're passing the ACA-Py model to the service. It's just about ensuring we're not passing extra fields which will be ignored -- safer to ensure we're passing acapy model to acapy.